bigquery_etl/util/bigquery_tables.py

```diff
@@ -36,7 +39,7 @@ def get_tables_matching_patterns(client: bigquery.Client, patterns):
                 datasets = [d for d in all_datasets[project] if fnmatchcase(d, dataset)]
             for dataset in datasets:
                 dataset = f"{project}.{dataset}"
-                tables = [(f"{dataset}.{table}", None)]
+                tables = [f"{dataset}.{table}"]
```

Comment:
r? @scholtzan 

I checked the usage of this function because `mypy` was complaining about inconsistent typing on `tables`, and it looks like this isn't supposed to be a `Tuple`?